### ISSUE TITLE
Weird noise in 0.17.1 build output

### ISSUE THREAD

**Issue by smajl** [ionic-team/stencil#1353]
**Stencil version:**

```
 @stencil/core@<0.17.1>
```

**I'm submitting a:**

[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

**Current behavior:**
Build output spills out stencil config, so it looks like:
```
> stencil build --dev --watch --serve

"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
var postcss_1 = require("@stencil/postcss");
var autoprefixer_1 = __importDefault(require("autoprefixer"));
// @ts-ignore
var postcss_nested_1 = __importDefault(require("postcss-nested"));
exports.config = {
    namespace: 'xyz',
    outputTargets: [
        { type: 'dist' },
        { type: 'docs' },
        {
            type: 'www',
            serviceWorker: null,
        },
    ],
    plugins: [
        ...
    ],
};

[38:20.7]  @stencil/core v0.17.1 
...
```

**Comment by vogloblinsky** [comment]
I think it is comming from this commit : https://github.com/ionic-team/stencil/commit/e7de19f364e2d11385275da548236fad2a97ed51 + https://github.com/ionic-team/stencil/blob/master/src/sys/node/node-config.ts#L98 from @manucorporat

**Comment by adamdbradley** [comment]
Should be fixed in 0.17.2, thanks
